apps/server/src/modules/server/server.config.ts

```diff
@@ -229,6 +231,7 @@ const config: ServerConfig = {
 	FEATURE_OTHER_GROUPUSERS_PROVISIONING_ENABLED: Configuration.get(
 		'FEATURE_OTHER_GROUPUSERS_PROVISIONING_ENABLED'
 	) as boolean,
+	ALERT_CACHE_INTERVAL: Configuration.get('ALERT_CACHE_INTERVAL') as number,
```

Comment:
When you add the configuration inside the module, then you should not do it twice on this place. Use ...getAlertConfig() instead. But i expect that you get a collision with SC_THEME. Also maybe you remove getAlertConfig completly inside the altert module.